esp/bindings/http/client/httpclient.cpp

```diff
@@ -50,7 +50,31 @@ CHttpClientContext::CHttpClientContext(IPropertyTree* config) : m_config(config)
 
 void CHttpClientContext::initPersistentHandler()
 {
-    m_persistentHandler.setown(createPersistentHandler(nullptr, DEFAULT_MAX_PERSISTENT_IDLE_TIME, DEFAULT_MAX_PERSISTENT_REQUESTS, static_cast<PersistentLogLevel>(getEspLogLevel()), true));
+    IEspServer* server = queryEspServer();
+    IPropertyTree* proc_cfg = nullptr;
+    if (server)
+        proc_cfg = server->queryProcConfig();
+    const char* idleTimeStr = nullptr;
+    const char* maxReqsStr = nullptr;
+    if (proc_cfg != nullptr)
```

Comment:
Could just use?:
int maxIdleTime = DEFAULT_MAX_PERSISTENT_IDLE_TIME;
int maxReqs = DEFAULT_MAX_PERSISTENT_REQUESTS;
if (proc_cfg)
{
    maxIdleTime = proc_cfg->queryPropInt("@maxPersistentIdleTime", maxIdleTime);
    maxReqs = proc_cfg->queryPropInt("@maxPersistentRequests", maxReqs);
}

Reply:
Good point, let me make the change.